examples/practice-typescript/src/components/SideBarMenuItem/sideBarMenuItem.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../styles/bases/variable.scss';
+
+.aside__item {
+  flex-direction: column;
+  justify-content: center;
+  height: 60px;
+  text-decoration: none;
+  display: flex;
```

Comment:
display: flex should come first, and before everything else that related to flex like flex-direction, justify-content

Reply:
Fixed